stable/mcrouter/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+name: mcrouter
+home: https://github.com/facebook/mcrouter
+version: 0.1.0
+appVersion: 1.0
+description: Mcrouter is a memcached protocol router for scaling memcached deployments.
+sources:
+- https://github.com/facebook/mcrouter
+- https://github.com/jamescarr/mcrouter/
```

Comment:
What is this second repo for?

Reply:
Oh, I actually meant to add this link instead: https://github.com/jamescarr/mcrouter-docker

Facebook provides a Dockerfile (https://github.com/facebook/mcrouter/blob/master/mcrouter/scripts/docker/Dockerfile). However, it unfortunately doesn't maintain an official image. Instead, the official wiki suggests using the jamescarr third-party image (https://github.com/facebook/mcrouter/wiki/mcrouter-installation#third-party-dockers).

Reply:
@viglesiasce  Alternatively, would it be acceptable for me to publish my own image with a more recent version of mcrouter, perhaps?

Reply:
Note that I'm calling this out in the README and encourage the user to create and use their own image: https://github.com/jphalip/charts/tree/mcrouter/stable/mcrouter#configuration